.env

```diff
@@ -0,0 +1 @@
+API_KEY=a99996dd4383ac814120045852e03bd6
```

Comment:
This file should never be pushed to GitHub, please make sure to add the `.env` file to the `.gitignore`.